Auto update failing for taskwarrior

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [taskwarrior](https://github.com/termux/termux-packages/blob/ebd338aeac70a7ec2acc25e1ff21be6b03409c66/packages/taskwarrior) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating taskwarrior [Current version: 3.4.0]
INFO: package being updated to 3.4.1.
Downloading https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v3.4.1/task-3.4.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  925k  100  925k    0     0  1485k      0 --:--:-- --:--:-- --:--:-- 1485k

You are about to commit these changes:

--------------------
diff --git a/packages/taskwarrior/build.sh b/packages/taskwarrior/build.sh
index 6d391f0ae0..f7ff8d1387 100644
--- a/packages/taskwarrior/build.sh
+++ b/packages/taskwarrior/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://taskwarrior.org
 TERMUX_PKG_DESCRIPTION="Utility for managing your TODO list"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="3.4.0"
+TERMUX_PKG_VERSION="3.4.1"
 TERMUX_PKG_SRCURL=https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v${TERMUX_PKG_VERSION}/task-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=8a4b1a062ff656e31c8abf842da9f1bfe8189102ce9240bf730e92378d52ecc2
+TERMUX_PKG_SHA256=23eb60f73e42f16111cc3912b44ee12be6768860a2db2a9c6a47f8ac4786bac3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libandroid-glob, libc++, libgnutls, libuuid"
 TERMUX_CMAKE_BUILD="Unix Makefiles"
--------------------

bump(main/taskwarrior): 3.4.1


Not committing to Git!

INFO: Trying to build package.
INFO: Package taskwarrior exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building taskwarrior for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  55302      0 --:--:-- --:--:-- --:--:-- 55417
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1447      0 --:--:-- --:--:-- --:--:--  1448
gpg: Signature made Fri 14 Mar 2025 09:01:39 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24524      0 --:--:-- --:--:-- --:--:-- 24485
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1506      0 --:--:-- --:--:-- --:--:--  1506
100   833  100   833    0     0   1506      0 --:--:-- --:--:-- --:--:--  1506
gpg: Signature made Fri 14 Mar 2025 12:26:56 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  23075      0 --:--:-- --:--:-- --:--:-- 23098
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1443      0 --:--:-- --:--:-- --:--:--  1441
100   833  100   833    0     0   1442      0 --:--:-- --:--:-- --:--:--  1441
gpg: Signature made Fri 14 Mar 2025 02:27:05 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libandroid-glob@0.6-2 if necessary...
Found libandroid-glob in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-glob to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2025.02.25 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2025.02.25 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency libgmp@6.3.0-1 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgmp/libgmp_6.3.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  320k  100  320k    0     0  5414k      0 --:--:-- --:--:-- --:--:-- 5431k
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.18 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.18 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunistring@1.3 if necessary...
Found libunistring in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunistring/libunistring_1.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  535k  100  535k    0     0  9428k      0 --:--:-- --:--:-- --:--:-- 9561k
extracting libunistring to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libidn2@2.3.7 if necessary...
Found libidn2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libidn2/libidn2_2.3.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  102k  100  102k    0     0  2060k      0 --:--:-- --:--:-- --:--:-- 2097k
extracting libidn2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnettle@3.10.1 if necessary...
Found libnettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnettle/libnettle_3.10.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  397k  100  397k    0     0  6715k      0 --:--:-- --:--:-- --:--:-- 6744k
extracting libnettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency nettle@3.10.1 if necessary...
Found nettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/nettle/nettle_3.10.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 18176  100 18176    0     0   348k      0 --:--:-- --:--:-- --:--:--  348k
100 18176  100 18176    0     0   347k      0 --:--:-- --:--:-- --:--:--  341k
extracting nettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libevent@2.1.12-2 if necessary...
Found libevent in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libevent/libevent_2.1.12-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  194k  100  194k    0     0  3002k      0 --:--:-- --:--:-- --:--:-- 3042k
extracting libevent to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.7.0 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.65.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.4.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.4.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gdbm/gdbm_1.24_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  139k  100  139k    0     0  2514k      0 --:--:-- --:--:-- --:--:-- 2544k
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-posix-semaphore/libandroid-posix-semaphore_0.1-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4128  100  4128    0     0  58305      0 --:--:-- --:--:-- --:--:-- 58971
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.7 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30588  100 30588    0     0   616k      0 --:--:-- --:--:-- --:--:--  622k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.4 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.6.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  184k  100  184k    0     0  2673k      0 --:--:-- --:--:-- --:--:-- 2680k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.4 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.6.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 67036  100 67036    0     0   617k      0 --:--:-- --:--:-- --:--:--  617k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.49.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.49.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.49.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.47 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.16.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.16.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.12 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.6.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.12 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.12 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.9 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python/python_3.12.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 42 7417k   42 3134k    0     0  39.0M      0 --:--:-- --:--:-- --:--:-- 38.7M
100 7417k  100 7417k    0     0  63.7M      0 --:--:-- --:--:-- --:--:-- 63.5M
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.9 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-tkinter/python-tkinter_3.12.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1191k  100 1191k    0     0  19.0M      0 --:--:-- --:--:-- --:--:-- 19.3M
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.9 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-ensurepip-wheels/python-ensurepip-wheels_3.12.9_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1740k  100 1740k    0     0  24.7M      0 --:--:-- --:--:-- --:--:-- 24.9M
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/resolv-conf/resolv-conf_1.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   976  100   976    0     0  22442      0 --:--:-- --:--:-- --:--:-- 22697
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.45 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.45 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency swig@4.3.0 if necessary...
Found swig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/swig/swig_4.3.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  7  909k    7 68902    0     0  1497k      0 --:--:-- --:--:-- --:--:-- 1495k
100  909k  100  909k    0     0  16.9M      0 --:--:-- --:--:-- --:--:-- 16.7M
extracting swig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunbound@1.22.0 if necessary...
Found libunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunbound/libunbound_1.22.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  351k  100  351k    0     0  6606k      0 --:--:-- --:--:-- --:--:-- 6638k
extracting libunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency unbound@1.22.0 if necessary...
Found unbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/u/unbound/unbound_1.22.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  578k  100  578k    0     0  10.6M      0 --:--:-- --:--:-- --:--:-- 10.6M
extracting unbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pyunbound@1.22.0 if necessary...
Found pyunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pyunbound/pyunbound_1.22.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39236  100 39236    0     0   849k      0 --:--:-- --:--:-- --:--:--  870k
extracting pyunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgnutls@3.8.9 if necessary...
Found libgnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgnutls/libgnutls_3.8.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  719k  100  719k    0     0  11.6M      0 --:--:-- --:--:-- --:--:-- 11.7M
extracting libgnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gnutls@3.8.9 if necessary...
Found gnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gnutls/gnutls_3.8.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  171k  100  171k    0     0  2863k      0 --:--:-- --:--:-- --:--:-- 2902k
extracting gnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libblkid@2.40.2-3 if necessary...
Found libblkid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libblkid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmount@2.40.2-3 if necessary...
Found libmount in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmount to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsmartcols@2.40.2-3 if necessary...
Found libsmartcols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsmartcols to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libuuid@2.40.2-3 if necessary...
Found libuuid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libuuid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libfdisk@2.40.2-3 if necessary...
Found libfdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libfdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcap-ng@2:0.8.5 if necessary...
Found libcap-ng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcap-ng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency util-linux@2.40.2-3 if necessary...
Found util-linux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting util-linux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency uuid-utils@2.40.2-3 if necessary...
Found uuid-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting uuid-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency mount-utils@2.40.2-3 if necessary...
Found mount-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting mount-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fdisk@2.40.2-3 if necessary...
Found fdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency blk-utils@2.40.2-3 if necessary...
Found blk-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting blk-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v3.4.1/task-3.4.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  4  925k    4 39649    0     0  96858      0  0:00:09 --:--:--  0:00:09 96858
100  925k  100  925k    0     0  2233k      0 --:--:-- --:--:-- --:--:--  173M
info: downloading installer
warn: It looks like you have an existing rustup settings file at:
warn: /home/builder/.rustup/settings.toml
warn: Rustup will install the default toolchain as specified in the settings file,
warn: instead of the one inferred from the default host triple.
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warn: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.84.1-x86_64-unknown-linux-gnu'

info: default toolchain set to '1.84.1-x86_64-unknown-linux-gnu'
  1.84.1-x86_64-unknown-linux-gnu unchanged - rustc 1.84.1 (e71f9a9a9 2025-01-27)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, you need to source
the corresponding env file under $HOME/.cargo.

This is usually done by running one of the following (note the leading DOT):
. "$HOME/.cargo/env"            # For sh/bash/zsh/ash/dash/pdksh
source "$HOME/.cargo/env.fish"  # For fish
source "$HOME/.cargo/env.nu"    # For nushell
info: component 'rust-std' for target 'aarch64-linux-android' is up to date
Downloading https://github.com/Kitware/CMake/releases/download/v3.31.6/cmake-3.31.6-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 52.4M  100 52.4M    0     0  69.2M      0 --:--:-- --:--:-- --:--:--  109M
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 18.0.3
-- The CXX compiler identification is Clang 18.0.3
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring C++17
-- System: Android
-- Performing Test _HAS_CXX17
-- Performing Test _HAS_CXX17 - Success
-- Looking for git submodules
-- Found git submodules
-- Looking for SHA1 references
-- Looking for timegm
-- Looking for timegm - found
-- Looking for get_current_dir_name
-- Looking for get_current_dir_name - not found
-- Looking for wordexp
-- Looking for wordexp - not found
-- Performing Test HAVE_TM_GMTOFF
-- Performing Test HAVE_TM_GMTOFF - Success
-- Performing Test HAVE_ST_BIRTHTIME
-- Looking for libuuid
-- Performing Test HAVE_ST_BIRTHTIME - Failed
-- Looking for uuid_unparse_lower
-- Looking for uuid_unparse_lower - found
-- Found libuuid
-- Configuring cmake.h
-- Rust Toolchain: 1.84.1-x86_64-unknown-linux-gnu
-- Rust Target: aarch64-linux-android
-- Determining required link libraries for target aarch64-linux-android
-- Required static libs for target aarch64-linux-android: dl;log;unwind;dl;m;c
-- Required link flags for target aarch64-linux-android: 
-- Determining required link libraries for target x86_64-unknown-linux-gnu
-- Required static libs for host target x86_64-unknown-linux-gnu: gcc_s;util;rt;pthread;m;dl;c
-- Found Rust: /home/builder/.rustup/toolchains/1.84.1-x86_64-unknown-linux-gnu/bin/rustc (found version "1.84.1")
-- Configuring man pages
-- Configuring done (4.9s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_INSTALL_LIBDIR
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Generating done (0.1s)
-- Build files have been written to: /home/builder/.termux-build/taskwarrior/build
[  1%] Building cxxbridge (version 1.0.144) with Rust toolchain 1.84.1-x86_64-unknown-linux-gnu
[  1%] Built target cargo-prebuild_taskchampion_lib
[  3%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Color.cpp.o
[  3%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Configuration.cpp.o
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on package cache
   Compiling libc v0.2.171
   Compiling proc-macro2 v1.0.94
   Compiling unicode-ident v1.0.18
   Compiling cfg-if v1.0.0
   Compiling autocfg v1.4.0
   Compiling shlex v1.3.0
[  3%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Datetime.cpp.o
   Compiling smallvec v1.14.0
   Compiling bytes v1.10.1
[  4%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Duration.cpp.o
   Compiling pin-project-lite v0.2.16
   Compiling itoa v1.0.15
   Compiling lock_api v0.4.12
   Compiling quote v1.0.40
   Compiling syn v2.0.100
   Compiling jobserver v0.1.32
   Compiling parking_lot_core v0.9.10
   Compiling cc v1.2.16
   Compiling futures-core v0.3.31
   Compiling scopeguard v1.2.0
   Compiling zeroize v1.8.1
[  5%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/FS.cpp.o
   Compiling parking_lot v0.12.3
   Compiling socket2 v0.5.8
[  6%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/JSON.cpp.o
   Compiling mio v1.0.3
   Compiling signal-hook-registry v1.4.2
   Compiling getrandom v0.2.15
   Compiling fnv v1.0.7
   Compiling once_cell v1.21.0
[  6%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Msg.cpp.o
   Compiling subtle v2.6.1
[  7%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Pig.cpp.o
   Compiling serde v1.0.219
   Compiling version_check v0.9.5
   Compiling pin-utils v0.1.0
   Compiling slab v0.4.9
   Compiling futures-sink v0.3.31
   Compiling http v1.3.1
   Compiling ring v0.17.14
   Compiling futures-task v0.3.31
   Compiling tracing-core v0.1.33
[  8%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/RX.cpp.o
[  9%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Table.cpp.o
   Compiling http-body v1.0.1
[  9%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/Timer.cpp.o
   Compiling num-traits v0.2.19
   Compiling typenum v1.18.0
   Compiling ryu v1.0.20
   Compiling untrusted v0.9.0
[ 10%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/format.cpp.o
   Compiling generic-array v0.14.7
[ 11%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/ip.cpp.o
[ 12%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/shared.cpp.o
   Compiling tokio-macros v2.5.0
   Compiling serde_derive v1.0.219
[ 12%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/unicode.cpp.o
   Compiling futures-macro v0.3.31
[ 13%] Building CXX object src/CMakeFiles/libshared.dir/libshared/src/utf8.cpp.o
   Compiling tokio v1.44.0
   Compiling tracing-attributes v0.1.28
   Compiling futures-util v0.3.31
[ 14%] Linking CXX static library liblibshared.a
[ 14%] Built target libshared
   Compiling tracing v0.1.41
   Compiling cmake v0.1.54
   Compiling time-core v0.1.3
   Compiling fs_extra v1.3.0
   Compiling foldhash v0.1.4
   Compiling powerfmt v0.2.0
   Compiling dunce v1.0.5
   Compiling num-conv v0.1.0
   Compiling time-macros v0.2.20
   Compiling tokio-util v0.7.13
   Compiling deranged v0.3.11
   Compiling aws-lc-sys v0.27.0
   Compiling num-integer v0.1.46
   Compiling http-body-util v0.1.3
   Compiling synstructure v0.13.1
   Compiling equivalent v1.0.2
   Compiling allocator-api2 v0.2.21
   Compiling time v0.3.39
   Compiling hashbrown v0.15.2
warning: aws-lc-sys@0.27.0: Building with: CMake
warning: aws-lc-sys@0.27.0: Symbol Prefix: Some("aws_lc_0_27_0")
warning: aws-lc-sys@0.27.0: CMAKE environment variable set: cmake
warning: aws-lc-sys@0.27.0: Preserving optimization flag: -Oz
warning: aws-lc-sys@0.27.0: Compilation of 'c11.c' succeeded - Ok(["/home/builder/.termux-build/taskwarrior/build/./cargo/build/aarch64-linux-android/release/build/aws-lc-sys-59540f79efd85c20/out/out-c11/7dfda64fdf5a526c-c11.o"]).
warning: aws-lc-sys@0.27.0: CC environment variable set: aarch64-linux-android-clang
warning: aws-lc-sys@0.27.0: CXX environment variable set: aarch64-linux-android-clang++
warning: aws-lc-sys@0.27.0: Using flag: -ffile-prefix-map=/home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/aws-lc-sys-0.27.0=
warning: aws-lc-sys@0.27.0: ANDROID_NDK_ROOT not set.
warning: aws-lc-sys@0.27.0: ANDROID_NDK not set.
warning: aws-lc-sys@0.27.0: ANDROID_STANDALONE_TOOLCHAIN not set.
error: failed to run custom build command for `aws-lc-sys v0.27.0`
note: To improve backtraces for build dependencies, set the CARGO_PROFILE_RELEASE_BUILD_OVERRIDE_DEBUG=true environment variable to enable debug information generation.

Caused by:
  process didn't exit successfully: `/home/builder/.termux-build/taskwarrior/build/./cargo/build/release/build/aws-lc-sys-72f7656c8fd2e4b4/build-script-main` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=AWS_LC_SYS_NO_PREFIX
  cargo:rerun-if-env-changed=AWS_LC_SYS_PREGENERATING_BINDINGS
  cargo:rerun-if-env-changed=AWS_LC_SYS_EXTERNAL_BINDGEN
  cargo:rerun-if-env-changed=AWS_LC_SYS_NO_ASM
  cargo:rerun-if-env-changed=AWS_LC_SYS_CFLAGS
  cargo:rerun-if-env-changed=AWS_LC_SYS_PREBUILT_NASM
  cargo:rerun-if-env-changed=AWS_LC_SYS_C_STD
  cargo:rustc-cfg=aarch64_linux_android
  cargo:rerun-if-env-changed=AWS_LC_SYS_CMAKE_BUILDER
  cargo:rerun-if-env-changed=AWS_LC_SYS_STATIC
  default_for Target: 'aarch64-linux-android'
  cargo:rerun-if-env-changed=AWS_LC_SYS_STATIC
  cargo:rerun-if-env-changed=CMAKE
  cargo:warning=Building with: CMake
  cargo:warning=Symbol Prefix: Some("aws_lc_0_27_0")
  cargo:rerun-if-env-changed=CMAKE
  cargo:warning=CMAKE environment variable set: cmake
  cargo:rerun-if-env-changed=AWS_LC_SYS_STATIC
  cargo:warning=Preserving optimization flag: -Oz
  cargo:rerun-if-env-changed=CMAKE_TOOLCHAIN_FILE_aarch64_linux_android
  cargo:rerun-if-env-changed=CMAKE_TOOLCHAIN_FILE
  OPT_LEVEL = Some(3)
  OUT_DIR = Some(/home/builder/.termux-build/taskwarrior/build/./cargo/build/aarch64-linux-android/release/build/aws-lc-sys-59540f79efd85c20/out)
  TARGET = Some(aarch64-linux-android)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CC_aarch64-linux-android
  CC_aarch64-linux-android = None
  cargo:rerun-if-env-changed=CC_aarch64_linux_android
  CC_aarch64_linux_android = Some(/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang)
  RUSTC_WRAPPER = None
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some(false)
  cargo:rerun-if-env-changed=CFLAGS
  CFLAGS = Some( -fstack-protector-strong -Oz --target=aarch64-linux-android24)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  cargo:rerun-if-env-changed=TARGET_CFLAGS
  TARGET_CFLAGS = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_linux_android
  CFLAGS_aarch64_linux_android = Some( -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz)
  cargo:rerun-if-env-changed=CFLAGS_aarch64-linux-android
  CFLAGS_aarch64-linux-android = None
  CARGO_ENCODED_RUSTFLAGS = Some(-L/data/data/com.termux/files/usr/lib-Clink-arg=-Wl,-rpath=/data/data/com.termux/files/usr/lib-Clink-arg=-Wl,--enable-new-dtags)
  OPT_LEVEL = Some(3)
  TARGET = Some(aarch64-linux-android)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CC_aarch64-linux-android
  CC_aarch64-linux-android = None
  cargo:rerun-if-env-changed=CC_aarch64_linux_android
  CC_aarch64_linux_android = Some(/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang)
  RUSTC_WRAPPER = None
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some(false)
  cargo:rerun-if-env-changed=CFLAGS
  CFLAGS = Some( -fstack-protector-strong -Oz --target=aarch64-linux-android24)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  cargo:rerun-if-env-changed=TARGET_CFLAGS
  TARGET_CFLAGS = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_linux_android
  CFLAGS_aarch64_linux_android = Some( -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz)
  cargo:rerun-if-env-changed=CFLAGS_aarch64-linux-android
  CFLAGS_aarch64-linux-android = None
  CARGO_ENCODED_RUSTFLAGS = Some(-L/data/data/com.termux/files/usr/lib-Clink-arg=-Wl,-rpath=/data/data/com.termux/files/usr/lib-Clink-arg=-Wl,--enable-new-dtags)
  cargo:warning=Compilation of 'c11.c' succeeded - Ok(["/home/builder/.termux-build/taskwarrior/build/./cargo/build/aarch64-linux-android/release/build/aws-lc-sys-59540f79efd85c20/out/out-c11/7dfda64fdf5a526c-c11.o"]).
  cargo:rerun-if-env-changed=CC
  cargo:warning=CC environment variable set: aarch64-linux-android-clang
  cargo:rerun-if-env-changed=CXX
  cargo:warning=CXX environment variable set: aarch64-linux-android-clang++
  OUT_DIR = Some(/home/builder/.termux-build/taskwarrior/build/./cargo/build/aarch64-linux-android/release/build/aws-lc-sys-59540f79efd85c20/out)
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  TARGET = Some(aarch64-linux-android)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CFLAGS
  CFLAGS = Some( -fstack-protector-strong -Oz --target=aarch64-linux-android24)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  cargo:rerun-if-env-changed=TARGET_CFLAGS
  TARGET_CFLAGS = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_linux_android
  CFLAGS_aarch64_linux_android = Some( -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz)
  cargo:rerun-if-env-changed=CFLAGS_aarch64-linux-android
  CFLAGS_aarch64-linux-android = None
  cargo:warning=Using flag: -ffile-prefix-map=/home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/aws-lc-sys-0.27.0=
  cargo:rerun-if-env-changed=ANDROID_NDK_ROOT
  cargo:warning=ANDROID_NDK_ROOT not set.
  cargo:rerun-if-env-changed=ANDROID_NDK
  cargo:warning=ANDROID_NDK not set.
  cargo:rerun-if-env-changed=ANDROID_STANDALONE_TOOLCHAIN
  cargo:warning=ANDROID_STANDALONE_TOOLCHAIN not set.
  CMAKE_TOOLCHAIN_FILE_aarch64-linux-android = None
  CMAKE_TOOLCHAIN_FILE_aarch64_linux_android = None
  TARGET_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_aarch64-linux-android = None
  CMAKE_GENERATOR_aarch64_linux_android = None
  TARGET_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_aarch64-linux-android = None
  CMAKE_PREFIX_PATH_aarch64_linux_android = None
  TARGET_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_aarch64-linux-android = None
  CMAKE_aarch64_linux_android = None
  TARGET_CMAKE = None
  CMAKE = Some("cmake")
  running: cd "/home/builder/.termux-build/taskwarrior/build/./cargo/build/aarch64-linux-android/release/build/aws-lc-sys-59540f79efd85c20/out/build" && CMAKE_PREFIX_PATH="" LC_ALL="C" "cmake" "/home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/aws-lc-sys-0.27.0" "-DBUILD_SHARED_LIBS=0" "-DBORINGSSL_PREFIX=aws_lc_0_27_0_" "-DBORINGSSL_PREFIX_HEADERS=/home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/aws-lc-sys-0.27.0/generated-include" "-DBUILD_TESTING=OFF" "-DBUILD_LIBSSL=OFF" "-DDISABLE_PERL=ON" "-DDISABLE_GO=ON" "-DCMAKE_C_STANDARD=11" "-DCMAKE_SYSTEM_NAME=Android" "-DCMAKE_SYSTEM_PROCESSOR=aarch64" "-DCMAKE_INSTALL_PREFIX=/home/builder/.termux-build/taskwarrior/build/./cargo/build/aarch64-linux-android/release/build/aws-lc-sys-59540f79efd85c20/out" "-DCMAKE_C_FLAGS= -Oz -ffile-prefix-map=/home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/aws-lc-sys-0.27.0= -DANDROID -ffunction-sections -fdata-sections -fPIC --target=aarch64-linux-android -fstack-protector-strong --target=aarch64-linux-android24 -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -fstack-protector-strong" "-DCMAKE_C_COMPILER=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang" "-DCMAKE_CXX_FLAGS= -DANDROID -ffunction-sections -fdata-sections -fPIC --target=aarch64-linux-android -fstack-protector-strong -Wno-c++11-narrowing --target=aarch64-linux-android24" "-DCMAKE_CXX_COMPILER=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++" "-DCMAKE_ASM_FLAGS= -DANDROID -ffunction-sections -fdata-sections -fPIC --target=aarch64-linux-android -fstack-protector-strong --target=aarch64-linux-android24 -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -fstack-protector-strong" "-DCMAKE_ASM_COMPILER=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang" "-DCMAKE_BUILD_TYPE=Release" "--no-warn-unused-cli"
  Not searching for unused variables given on the command line.
  -- Configuring incomplete, errors occurred!

  --- stderr
  CMake Deprecation Warning at CMakeLists.txt:10 (cmake_policy):
    The OLD behavior for policy CMP0091 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.


  CMake Error at /home/builder/.termux-build/_cache/cmake-3.31.6/share/cmake-3.31/Modules/Platform/Android-Determine.cmake:218 (message):
    Android: Neither the NDK or a standalone toolchain was found.
  Call Stack (most recent call first):
    /home/builder/.termux-build/_cache/cmake-3.31.6/share/cmake-3.31/Modules/CMakeDetermineSystem.cmake:184 (include)
    CMakeLists.txt:13 (project)


  CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
  thread 'main' panicked at /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.54/src/lib.rs:1119:5:

  command did not execute successfully, got: exit status: 1

  build script failed, must exit now
  stack backtrace:
     0:     0x5593093c17ca - <std::sys::backtrace::BacktraceLock::print::DisplayBacktrace as core::fmt::Display>::fmt::h14b23c1989cbd5c2
     1:     0x5593093e7823 - core::fmt::write::h346b5eee5ed4d7cc
     2:     0x5593093bd773 - std::io::Write::write_fmt::heeb48dcd4a60b46b
     3:     0x5593093c1612 - std::sys::backtrace::BacktraceLock::print::h48db11f3fd4983ff
     4:     0x5593093c280c - std::panicking::default_hook::{{closure}}::h65db976b9c0d8674
     5:     0x5593093c2652 - std::panicking::default_hook::h6eb3a1192db1ae36
     6:     0x5593093c2e47 - std::panicking::rust_panic_with_hook::h370ee1901241e459
     7:     0x5593093c2cda - std::panicking::begin_panic_handler::{{closure}}::heabfe92676d6d073
     8:     0x5593093c1ca9 - std::sys::backtrace::__rust_end_short_backtrace::h6e22d229d4fdf49e
     9:     0x5593093c296c - rust_begin_unwind
    10:     0x5593093e6670 - core::panicking::panic_fmt::hfae270fab21da3e6
    11:     0x559309305bf2 - cmake::fail::h5bfa5cd3252fe22a
    12:     0x55930930538d - cmake::run::h907112cef19b56a3
    13:     0x55930930150f - cmake::Config::build::h0df43c1b83624e5a
    14:     0x5593092f0ae0 - build_script_main::cmake_builder::CmakeBuilder::build_rust_wrapper::h6aba45bf4df14fb9
    15:     0x5593092f1025 - <build_script_main::cmake_builder::CmakeBuilder as build_script_main::Builder>::build::h050799f7a2826ea4
    16:     0x5593092d984c - build_script_main::main::h449aef03dacddf16
    17:     0x5593092e27c3 - core::ops::function::FnOnce::call_once::h83eb9bd7c5ffaf6c
    18:     0x5593092e1ae6 - std::sys::backtrace::__rust_begin_short_backtrace::h67e11b0b951879c5
    19:     0x5593092e1aa9 - std::rt::lang_start::{{closure}}::h7e4bbfa0b939b3dc
    20:     0x5593093b7a27 - std::rt::lang_start_internal::h712a1d4742291d0c
    21:     0x5593092e1a87 - std::rt::lang_start::h3e5253e7a480ea6b
    22:     0x5593092dcc95 - main
    23:     0x7f81113311ca - <unknown>
    24:     0x7f811133128b - __libc_start_main
    25:     0x5593092d1615 - _start
    26:                0x0 - <unknown>
warning: build failed, waiting for other jobs to finish...
make[2]: *** [src/taskchampion-cpp/CMakeFiles/_cargo-build_taskchampion_lib.dir/build.make:70: src/taskchampion-cpp/CMakeFiles/_cargo-build_taskchampion_lib] Error 101
make[1]: *** [CMakeFiles/Makefile2:473: src/taskchampion-cpp/CMakeFiles/_cargo-build_taskchampion_lib.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 14%] Built target cxxbridge_v1.0.144
make: *** [Makefile:166: all] Error 2
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-03-15 01:52:02 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/13867405715">13867405715</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/taskwarrior/build.sh
+++ b/packages/taskwarrior/build.sh
@@ -2,18 +2,21 @@ TERMUX_PKG_HOMEPAGE=https://taskwarrior.org
 TERMUX_PKG_DESCRIPTION="Utility for managing your TODO list"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="3.4.0"
+TERMUX_PKG_VERSION="3.4.1"
 TERMUX_PKG_SRCURL=https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v${TERMUX_PKG_VERSION}/task-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=8a4b1a062ff656e31c8abf842da9f1bfe8189102ce9240bf730e92378d52ecc2
+TERMUX_PKG_SHA256=23eb60f73e42f16111cc3912b44ee12be6768860a2db2a9c6a47f8ac4786bac3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libandroid-glob, libc++, libgnutls, libuuid"
 TERMUX_CMAKE_BUILD="Unix Makefiles"
 
 termux_step_pre_configure() {
 	termux_setup_rust
+	cargo install --force --locked bindgen-cli
 
 	CXXFLAGS+=" -Wno-c++11-narrowing"
 	LDFLAGS+=" -landroid-glob"
+	export ANDROID_STANDALONE_TOOLCHAIN="$TERMUX_STANDALONE_TOOLCHAIN"
+	export CARGO_TARGET_$(printf $CARGO_TARGET_NAME | tr a-z A-Z | sed s/-/_/g)_RUSTFLAGS+=" -C linker=$(command -v $CC)"
 
 	if [ "$TERMUX_ARCH" = "arm" ]; then
 		# See https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARM_MODE.html

